.eslintrc

```diff
@@ -54,7 +54,18 @@
     },
     {
       "files": ["**/*.ts", "**/*.tsx"],
-      "extends": ["plugin:instawork/recommended-typescript"]
+      "extends": ["plugin:instawork/recommended-typescript"],
+      "rules": {
```

Comment:
FYI @ashwinibm - we might want to move these into our aslant plugin at some point